android/src/main/java/com/oblador/storereview/StoreReviewModule.kt

```diff
@@ -0,0 +1,50 @@
+package com.oblador.storereview
+
+import android.os.Build.VERSION.SDK_INT
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.bridge.ReactContextBaseJavaModule
+import com.facebook.react.bridge.ReactMethod
+import com.facebook.react.bridge.Promise
+import com.google.android.play.core.review.ReviewManagerFactory
+
+class StoreReviewModule(reactContext: ReactApplicationContext) : ReactContextBaseJavaModule(reactContext) {
+
+    override fun getName(): String {
+        return "RNStoreReview"
+    }
+
+    override fun getConstants(): MutableMap<String, Any> {
+        return mutableMapOf<String, Any>("isAvailable" to (SDK_INT >= 21))
+    }
+
+    @ReactMethod
+    fun requestReview(promise: Promise) {
```

Comment:
The iOS implementation doesn't return a promise, is there any point to doing a promise here as it will never resolve to any value regardless?

Reply:
Nope, that was probably part of the empty bootstrap android project. Silly to keep it so it is removed.